ui/raidboss/raidboss_config.js

```diff
@@ -125,6 +125,7 @@ let kDetailKeys = {
       fr: 'Durée',
       ja: '存続時間',
       cn: '持续时间',
```

Comment:
Great! And you can also add unit to other languages too.

```javascript
      en: 'duration (in seconds)',
      de: 'Dauer (in Sekunden)',
      fr: 'Durée (en secondes)',
      ja: '存続時間 (秒)',
      cn: '持续时间 (秒)',
```

Reply:
do we need `in seconds` long? I think `sec` is enough and I like short thing.

Reply:
Well, I am not very familiar with grammar of English, I just copy them from the "hide cheese icon option".
Do you think it is normal to use `duration (sec)` in English?

Reply:
it's very normal to use `sec` `min` `hr` in English. I don't know French or German though.

Reply:
Maybe just `Sekunden` and `secondes` is fine.
Note: noun in German should be Capitalized always.

Reply:
Thanks! Having the time unit in there makes sense.